{"uid":"f87ab7f68634dd5b","name":"Login with valid credentials test","fullName":"tests.test_login.TestLogin#test_valid_login","historyId":"e7f254b9bfd5f2bc8d4a1ffa6dc7aebb","time":{"start":1705453520365,"stop":1705453530538,"duration":10173},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5593f108af83 <unknown>\n#1 0x5593f0d43cf7 <unknown>\n#2 0x5593f0d9399e <unknown>\n#3 0x5593f0d93aa1 <unknown>\n#4 0x5593f0dded64 <unknown>\n#5 0x5593f0dbd0dd <unknown>\n#6 0x5593f0ddc006 <unknown>\n#7 0x5593f0dbce53 <unknown>\n#8 0x5593f0d84dd4 <unknown>\n#9 0x5593f0d861de <unknown>\n#10 0x5593f104f531 <unknown>\n#11 0x5593f1053455 <unknown>\n#12 0x5593f103bf55 <unknown>\n#13 0x5593f10540ef <unknown>\n#14 0x5593f101f99f <unknown>\n#15 0x5593f1078008 <unknown>\n#16 0x5593f10781d7 <unknown>\n#17 0x5593f108a124 <unknown>\n#18 0x7f75ee694ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f187873ccd0>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"11aab95b7d920c5eba800015e90b9e6b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f18786a4860>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5593f108af83 <unknown>\nE       #1 0x5593f0d43cf7 <unknown>\nE       #2 0x5593f0d9399e <unknown>\nE       #3 0x5593f0d93aa1 <unknown>\nE       #4 0x5593f0dded64 <unknown>\nE       #5 0x5593f0dbd0dd <unknown>\nE       #6 0x5593f0ddc006 <unknown>\nE       #7 0x5593f0dbce53 <unknown>\nE       #8 0x5593f0d84dd4 <unknown>\nE       #9 0x5593f0d861de <unknown>\nE       #10 0x5593f104f531 <unknown>\nE       #11 0x5593f1053455 <unknown>\nE       #12 0x5593f103bf55 <unknown>\nE       #13 0x5593f10540ef <unknown>\nE       #14 0x5593f101f99f <unknown>\nE       #15 0x5593f1078008 <unknown>\nE       #16 0x5593f10781d7 <unknown>\nE       #17 0x5593f108a124 <unknown>\nE       #18 0x7f75ee694ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1705453520364,"stop":1705453520364,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1705453520364,"stop":1705453520364,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"secret_data","time":{"start":1705453520364,"stop":1705453520365,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1705453520365,"stop":1705453520365,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5593f108af83 <unknown>\n#1 0x5593f0d43cf7 <unknown>\n#2 0x5593f0d9399e <unknown>\n#3 0x5593f0d93aa1 <unknown>\n#4 0x5593f0dded64 <unknown>\n#5 0x5593f0dbd0dd <unknown>\n#6 0x5593f0ddc006 <unknown>\n#7 0x5593f0dbce53 <unknown>\n#8 0x5593f0d84dd4 <unknown>\n#9 0x5593f0d861de <unknown>\n#10 0x5593f104f531 <unknown>\n#11 0x5593f1053455 <unknown>\n#12 0x5593f103bf55 <unknown>\n#13 0x5593f10540ef <unknown>\n#14 0x5593f101f99f <unknown>\n#15 0x5593f1078008 <unknown>\n#16 0x5593f10781d7 <unknown>\n#17 0x5593f108a124 <unknown>\n#18 0x7f75ee694ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7f187873ccd0>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"11aab95b7d920c5eba800015e90b9e6b\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f18786a4860>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5593f108af83 <unknown>\nE       #1 0x5593f0d43cf7 <unknown>\nE       #2 0x5593f0d9399e <unknown>\nE       #3 0x5593f0d93aa1 <unknown>\nE       #4 0x5593f0dded64 <unknown>\nE       #5 0x5593f0dbd0dd <unknown>\nE       #6 0x5593f0ddc006 <unknown>\nE       #7 0x5593f0dbce53 <unknown>\nE       #8 0x5593f0d84dd4 <unknown>\nE       #9 0x5593f0d861de <unknown>\nE       #10 0x5593f104f531 <unknown>\nE       #11 0x5593f1053455 <unknown>\nE       #12 0x5593f103bf55 <unknown>\nE       #13 0x5593f10540ef <unknown>\nE       #14 0x5593f101f99f <unknown>\nE       #15 0x5593f1078008 <unknown>\nE       #16 0x5593f10781d7 <unknown>\nE       #17 0x5593f108a124 <unknown>\nE       #18 0x7f75ee694ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1705453520365,"stop":1705453530537,"duration":10172},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5593f108af83 <unknown>\n#1 0x5593f0d43cf7 <unknown>\n#2 0x5593f0d9399e <unknown>\n#3 0x5593f0d93aa1 <unknown>\n#4 0x5593f0dded64 <unknown>\n#5 0x5593f0dbd0dd <unknown>\n#6 0x5593f0ddc006 <unknown>\n#7 0x5593f0dbce53 <unknown>\n#8 0x5593f0d84dd4 <unknown>\n#9 0x5593f0d861de <unknown>\n#10 0x5593f104f531 <unknown>\n#11 0x5593f1053455 <unknown>\n#12 0x5593f103bf55 <unknown>\n#13 0x5593f10540ef <unknown>\n#14 0x5593f101f99f <unknown>\n#15 0x5593f1078008 <unknown>\n#16 0x5593f10781d7 <unknown>\n#17 0x5593f108a124 <unknown>\n#18 0x7f75ee694ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"61fedd4b35373e45","name":"public ip address","source":"61fedd4b35373e45.txt","type":"text/plain","size":13},{"uid":"855bb2c74ced565f","name":"Cookies","source":"855bb2c74ced565f.json","type":"application/json","size":2},{"uid":"14fbc19e0f4cadc6","name":"Session Storage","source":"14fbc19e0f4cadc6.json","type":"application/json","size":2},{"uid":"b56fbcc9a0c80a7","name":"Local Storage","source":"b56fbcc9a0c80a7.json","type":"application/json","size":2},{"uid":"c93f0db38c935512","name":"Console Logs","source":"c93f0db38c935512.json","type":"application/json","size":2},{"uid":"e3f930bf0ce874a4","name":"Full Page Screenshot","source":"e3f930bf0ce874a4.png","type":"image/png","size":2655658},{"uid":"2e4b27ae64d68057","name":"URL","source":"2e4b27ae64d68057.uri","type":"text/uri-list","size":25},{"uid":"f152a70614b46330","name":"Network Logs","source":"f152a70614b46330.json","type":"application/json","size":14400}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"tag","value":"Tagged test"},{"name":"severity","value":"blocker"},{"name":"feature","value":"Login"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az568-960"},{"name":"thread","value":"2170-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","run(order=1)"]},"source":"f87ab7f68634dd5b.json","parameterValues":[]}